timelock-server/src/test/java/com/palantir/atlasdb/timelock/lock/BlockingTimeLimitedLockServiceTest.java

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright 2017 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.timelock.lock;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.concurrent.Callable;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Test;
+
+import com.google.common.collect.ImmutableSortedMap;
+import com.google.common.util.concurrent.FakeTimeLimiter;
+import com.google.common.util.concurrent.TimeLimiter;
+import com.google.common.util.concurrent.UncheckedTimeoutException;
+import com.palantir.lock.LockClient;
+import com.palantir.lock.LockMode;
+import com.palantir.lock.LockRequest;
+import com.palantir.lock.LockService;
+import com.palantir.lock.StringLockDescriptor;
+import com.palantir.lock.remoting.BlockingTimeoutException;
+
+public class BlockingTimeLimitedLockServiceTest {
+    private static final long BLOCKING_TIME_LIMIT_MILLIS = 10L;
+    private static final long TIME = 42L;
```

Comment:
Sure, should be consistent with the above (and that's more descriptive with what it's used for as well)